src/backend/tests/test-data/projects.test-data.ts

```diff
@@ -162,7 +162,8 @@ export const prismaMaterialType: PrismaMaterialType = {
 };
 
 export const prismaUnit: Unit = {
-  name: 'FT'
+  name: 'FT',
+  dateDeleted: null
```

Comment:
Is this necessary if the date deleted field is optional?

Reply:
It required me to do it. Otherwise, the build failed.